operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServersUpStepTest.java

```diff
@@ -478,7 +489,15 @@ void withStartPolicyAlways_addNonManagedServers() {
   void whenShuttingDownAtLeastOneServer_prependServerDownIteratorStep() {
     addServer(info, "server1");
 
-    assertThat(firstNonEventStep(createNextStep()), instanceOf(ServerDownIteratorStep.class));
+    assertThat(firstNonEventAndNonUpdateCompleteConditionStep(createNextStep()),
```

Comment:
Again, try to test the behavioral change rather than the steps created to effect that behavior.